README.md

```diff
@@ -1,59 +1,54 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# School library
+
+> This app is a tool one can use in a library to record what books are in the library and who borrows them. This app will allow a user to:
+
+- Add new students or teachers.
+- Add new books.
+- Save records of who borrowed a book and when.
 
-> One paragraph statement about the project.
 
-![screenshot](./app_screenshot.png)
 
 Additional description about the project and its features.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Ruby
 
-## Live Demo
 
-[Live Demo Link](https://livedemo.com)
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
 
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
+```
+Ruby
+```
 
 ### Setup
+```
+git clone git@github.com:Tchilo/morse-code.git
+cd morse-code.git
+```
 
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
+### Run Program
+```
+ruby decode_morse_code.rb
+```
```

Comment:
- [ ] Your README here reflects documentation for Morse Code whereas it should be for School Library. Kindly fix this.